keras/layers/core/dense.py

```diff
@@ -105,9 +118,18 @@ def build(self, input_shape):
             self.bias = None
         self.input_spec = InputSpec(min_ndim=2, axes={-1: input_dim})
         self.built = True
+        if self.lora_rank:
+            self.enable_lora(self.lora_rank)
 
     def call(self, inputs):
-        x = ops.matmul(inputs, self.kernel)
+        if self.lora_enabled:
+            kernel = self.kernel + ops.matmul(
```

Comment:
Note: I get the feeling it will be better to have a `kernel` property that can be overridden.